setup.py

```diff
@@ -1,26 +1,23 @@
-from setuptools import setup
-import sys
 
-if not sys.version_info[0] == 3 and sys.version_info[1] < 8:
```

Comment:
Why delete this check?

Reply:
https://docs.astral.sh/ruff/rules/negate-equal-op/